velociraptor.ts

```diff
@@ -1,27 +1,28 @@
 import { ScriptsConfiguration } from "./dev_deps.ts";
 
+const common: Partial<ScriptsConfiguration> = {
+  allow: ["env", "read", "write"],
+  unstable: true,
+  watch: false,
+  imap: "./import_map.json",
+};
+
 export default <ScriptsConfiguration> {
   scripts: {
     install_docker_aliases: {
       cmd: "deno run scripts/install_aliases.ts docker",
```

Comment:
what do you think about use `deno run` as a variable?

---
**Sema Reaction:** :question: I have a question

Reply:
Do you mean to change it as a shell alias? Can you elaborate?